h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -103,6 +109,71 @@ public void initMetalearnerParams(Metalearner.Algorithm algo) {
     public Key<Frame>[] _base_model_predictions_keys; 
   }
 
+  public static class StackedEnsembleMRScorer extends MRTask<StackedEnsembleMRScorer> {
+    // IN
+    private Frame scoringFrame;
+    private String seKey;
+    private Job job;
+
+    // OUT
+    private Frame levelOneFrame;
```

Comment:
`MRTask` is a distributed task, therefore its attribute will be serialized, that's why we should be careful with those attributes types:
- no Frame, but use Key<Frame> instead: the `Frame` should be put in DKV, and retrieved the same way, and eventually removed after usage (for eample in reduce, you have to update the lhs in DKV, and you should remove the rhs).
- Job is also a `Keyed`, so even if it's a much smaller object than Frame, it's better to store just `Key<Job>` in those tasks as well (also note that job can be null).
- don't use String to store keys (`seKey`), but `Key<StackedEnsembleModel>`.

Reply:
also, naming conventions: I know they're not standard and can be annoying sometimes, but they are what they are... and let's be consistent in the same class: `_jobKey`, `_scoringFrameKey`, `_seKey`, `_levelOneFrameKey`